products/fairphone.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /fairphone
+title: Fairphone
+category: device
+releasePolicyLink: https://forum.fairphone.com/t/fairphone-product-release-cycle/52652
+discontinuedColumn: true
+activeSupportColumn: false
+eolColumn: Supported
+releaseColumn: false
+releaseDateColumn: true
+releases:
+-   releaseCycle: "Fairphone 4"
+    discontinued: false
+    eol: false
+    releaseDate: 2021-09-30
+-   releaseCycle: "Fairphone 3+"
+    discontinued: 2021-09-01
+    eol: 2021-09-01
+    releaseDate: 2020-09-30
+-   releaseCycle: "Fairphone 3"
+    discontinued: 2021-09-01
+    eol: 2021-09-01
+    releaseDate: 2019-09-30
+-   releaseCycle: "Fairphone 2"
+    discontinued: 2023-03-01
+    eol: 2023-03-01
+    releaseDate: 2015-12-01
+-   releaseCycle: "Fairphone 1"
+    discontinued: 2017-07-13
+    eol: 2017-07-13
+    releaseDate: 2013-12-01
+---
+
+> Fairphone is a Dutch electronics manufacturer that designs and produces smartphones with the goal of having a lower environmental footprint and better social impact than is common in the industry.
+
+> We aim to support Fairphone 4 with security and software upgrades for approximately five years from its launch
```

Comment:
The tone should always be third-person neutral present tense (See Style Guide on the wiki). "we" here is the Fairphone company, and we can't write on their behalf, unless we're quoting, and we should avoid it there as well to avoid confusion.

Something like:

Fairphone 4 is supported with security and software upgrades for approximately 5 years from its launch.

would be better.

However, this will get covered by the table already, so probably better to skip this line.
```suggestion
```